src/JsonService.ts

```diff
@@ -74,7 +122,12 @@ export class JsonService {
         return json;
     }
 
-    public async postForm(url: string, body: URLSearchParams, basicAuth?: string): Promise<Record<string, unknown>> {
+    public async postForm({
+        url,
```

Comment:
I don't have a particularly strong opinion on the API here, but I think keeping `url: string` for the first arguments would be nice it that it mirrors `fetch` and would make many of the tests simpler to write.